modules/k8s-nmstate-uninstall-operator.adoc

```diff
@@ -0,0 +1,212 @@
+----
+
+. If a `CatalogSource` resource exists, delete the resource by running the following command:
++
+[source,terminal]
+----
+$ oc delete --namespace openshift-marketplace catalogsource kubernetes-nmstate-catalog
+----
+
+. Determine if an `nmstate` CR, `Pod` resources, and `Deployment` resources exist by running the following commands:
++
+[source,terminal]
+----
+$ oc -n openshift-nmstate get nmstate
+----
++
+.Example output
+[source,terminal]
+----
+NAME  	  AGE
+nmstate   2d6h
+...
+----
++
+[source,terminal]
+----
+oc -n openshift-nmstate get deployment
+----
++
+.Example output
+[source,terminal]
+----
+NAME                 	 READY   UP-TO-DATE   AVAILABLE   AGE
+nmstate-console-plugin   0/1 	 1        	  0       	  2d5h
+nmstate-metrics      	 1/1 	 1        	  1       	  2d5h
+nmstate-webhook      	 2/2 	 2        	  2       	  2d5h
+----
++
+[source,terminal]
+----
+$ oc -n openshift-nmstate get pod
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                  	 READY   STATUS         	RESTARTS   AGE
+nmstate-console-plugin-867cccd945-sszr2  0/1 	 ImagePullBackOff   0      	   2d5h
+nmstate-handler-29qpd                 	 1/1 	 Running        	0      	   2d5h
+nmstate-handler-745fn                 	 1/1 	 Running        	0      	   2d5h
+...
+----
+
+. Delete the `nmstate` CR. After you delete the CR, any associated `Deployment` and `Pod` resources also get deleted.
++
+[source,terminal]
+----
+$ oc -n openshift-nmstate delete nmstate nmstate
+----
+
+. Optional: Identify if you need to delete or keep the NNCP manifest file. Consider that your cluster might depend on the NNCP for a specific postinstallation network configuration.
++
+.. Find any policy files and CRDs associated with your cluster by running the following commands:
++
+[source,terminal]
+----
+$ oc get nncp
+----
++
+[source,terminal]
+----
+NAME                      	  STATUS  	  REASON
+test-policy-2-additional-ip   Available   SuccessfullyConfigured
+test-policy-3-correct-dns 	  Available   SuccessfullyConfigured
+...
+----
++
+[source,terminal]
+----
+$ oc get crd | grep nmstate.io
+----
++
+.Example output
+[source,terminal]
+----
+nmstates.nmstate.io    2025-01-22T08:32:12Z   nodenetworkconfigurationenactments.nmstate.io                 	
+...
+----
++
+.. Delete the CRD and any NNCP associated with the Kubernetest NMState Operatgor.
++
+[IMPORTANT]
+====
+Deleting an NNCP does not delete any created objects, such as a bridge interface. To completely delete the NNCP, see The "NodeNetworkConfigurationPolicy manifest file" in the _Additional resources_ section.
+====
++
+[source,terminal]
+----
+$ oc delete crd nodenetworkconfigurationpolicies.nmstate.io
+----
++
+.Example output
+[source,terminal]
+----
+customresourcedefinition.apiextensions.k8s.io "nodenetworkconfigurationpolicies.nmstate.io" deleted
+----
+
+. Delete any remaining CRDs:
++
+[source,terminal]
+----
+$ oc get crd | grep nmstate.io
+...
+----
++
+.Example output
```

Comment:
Is the example output important?

Reply:
Not at all. You can go directly to removing, i.e.

```
oc delete crd nmstates.nmstate.io
oc delete crd nodenetworkconfigurationenactments.nmstate.io
oc delete crd nodenetworkstates.nmstate.io
oc delete crd nodenetworkconfigurationpolicies.nmstate.io
```